public/js/getEvents.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Gets event information from the API and passes into the displayHandler function
+ *
+ * @return event data
+ */
+async function getEvents() {
+    await fetch('/api/getEvents', {
+        credentials: "same-origin",
+        headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json',
+        },
+    })
+        .then(eventInfo => eventInfo.json())
+        .then(eventInfo => displayEventsHandler(eventInfo.data))
+}
+
+/**
+ * Runs a foreach through each event object and outputs HTML elements with event details
+ *
+ * @param events is an array of objects which contains information about events
+ *
+ * @return a divs of the event name with a button that reveals each event's additional info on each line
```

Comment:
should this be 'div'? Doesn't make sense to me.

Reply:
It doesn't return anything, it spits out html